node.go

```diff
@@ -270,7 +231,6 @@ func (node *Node) UnmarshalJSON(data []byte) error {
 }
 
 func (node Node) Equals(other Node) bool {
-	// TODO: add generatored code for this
```

Comment:
should open a github issue if you still care, although I think it's fine as it is